.github/actions/setup-localstack/action.yaml

```diff
@@ -20,9 +20,9 @@ runs:
         password: ${{ inputs.password }}
     - shell: bash
       run: |
-            pip install localstack==2.2.0                                # Install LocalStack cli
+            pip install localstack==2.3.1                                # Install LocalStack cli
             # Below image was copied manually from localstack/localstack:2.2 and uploaded to ghcr
```

Comment:
Update the comment too (maybe without tag).
The ideea of the comment is to know after a while where is the original image on docker hub